apiary.apib

```diff
@@ -778,6 +778,12 @@ Get details for a single company.
             + added_at: `2016-02-04T16:44:33+00:00` (string)
             + updated_at: `2016-02-05T16:44:33+00:00` (string)
             + tags: prospect, expo (array[string])
+            + custom_fields (array)
+                + (object)
+                    + field: (object)
+                        + type: `customField` (string)
```

Comment:
If we go for `customFieldDefinition` endpoints, we should also name this `customFieldDefinition`. @stivni

Reply:
This is how it was implemented, but just as optional include. I think it was needed for @mrtus 

I think we should also include more information here like the name of the custom field. This will the returned information a lot more logical and useful.

Reply:
Please don't break my custom fields 🙃  (I don't mind about the type though, we don't use that)

Reply:
@mrtus adding more properties like the name of the custom field couldn't really hurt, right?

Reply:
Offcourse not :) adding should never break an API

Reply:
@WouterSioen We can't embed data inside that `customFieldDefinition`. The `type` field of the  definition would conflict with the type already used here. For relations we return a type/id "tuple" that represents the relationship, but it will never hold data.

If we want to include information about the custom fields, we should side-load it and put it at the bottom of the page. Which in the end is the same as doing a `customFieldDefinitions.list` call with a filter.